templates/index.html

```diff
@@ -1,6 +1,46 @@
 {% extends 'base.html' %}
 
+{% block navbar %}
+
+<div class="row">
```

Comment:
Hi Phoebe,

The navbar needs to be in the base.html template as it is a common component across _all_ the pages.

Also, I'm not sure why we have a row class on the parent div of the navbar, considering that navbars from Bootstrap are Responsive by default and take up the full width of the viewport. Consider removing the row parent div?